test/README.md

```diff
@@ -0,0 +1,67 @@
+Candid test data
+================
+
+This directory contains Candid test data, to be used to check Candid
+implementations for compliance.
+
+This test suite covers
+
+ * parsing Candid service descriptions
+ * given a Candid type, decoding binary Candid data, asserting success or failure
+ * given a Candid type, decoding textual Candid data, asserting success or failure
```

Comment:
What does it mean to "decode a textual data", parsing? Ah, maybe you mean encode the text value at the given type.

Reply:
I do mean parsing.
```suggestion
 * given a Candid type, parsing textual Candid data at that type, asserting success or failure
```